### ISSUE TITLE
Socket timeout while opening Blueprint FTP links

### ISSUE THREAD

**Issue by iromeo** [JetBrains-Research/jbr#40]
Socket timeout while opening Blueprint FTP links.
E.g. try `ftp://ftp.ebi.ac.uk/pub/databases/blueprint/blueprint_progenitor_methylomes/rna/RNA_D1_CLP_100.bw`

Not clear why it happening. But the problem is in `htsjdk`. They implement it's own FTP client, which hangs while logging in to blueprint FTP server ( https://github.com/samtools/htsjdk/issues/797).
Apache FTPClient works ok.

**Comment by dievsky** [comment]
Is this issue still relevant?

**Comment by iromeo** [comment]
@dievsky Yes, still doesn't work for me. And does it work on your machine?

**Comment by dievsky** [comment]
It doesn't work, but not because of the socket timeout. The exception that I get is
```
java.lang.UnsupportedOperationException: read() is not supported on SeekableHTTPStream.  Must read in blocks.
```
when trying to "read one byte from url to be 100% sure". This is from `URIExtensions.kt:108`.

**Comment by dievsky** [comment]
(I just wanted to check up on an issue that has been dormant for almost 3 years.)
